### PR TITLE
Update for Azure

### PR DESCRIPTION
**Pull request by cdavid**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -4,9 +4,13 @@ NAME=ybm
 VERSION=0.1.0-dev
 OS_ARCH=darwin_arm64
 BINARY=terraform-provider-${NAME}
+export GOPRIVATE := github.com/yugabyte
 
 default: install
 
+vet:
```
Comment (shinji62) [inline Makefile:11]: did you rebase your branch ??

Comment (cdavid) [reply Makefile:11]: This is the Terraform provider - it never had a vet command in the `Makefile`.

Comment (shinji62) [reply Makefile:11]: Oh... that;s why the code was a bit different lol Sorry

**Review by posriniv — APPROVED** [review]
Looks good to me!

**Pull request merged**